### ISSUE TITLE
Unable to boot into gsi

### ISSUE THREAD

**Issue by Tahirpatel12345** [VegaBobo/DSU-Sideloader#60]
I have samsung a31 rooted and running on oneui4.1 stock rom i tried with all version of dsu sideloader and many gsi but sometimes device softboots when the dsu sideloader is rpcessibg userdata and everytime it boots to stock rom after clickinh restart option in notification

[logs.txt](https://github.com/VegaBobo/DSU-Sideloader/files/10801754/logs.txt)

**Comment by javierchip** [comment]
i have the same problem :(

**Comment by LeOS-GSI** [comment]
In most cases you have to flash a empty vbmeta to get booting into the DSU GSI

**Comment by VegaBobo** [comment]
Yes, i think also it is a vbmeta issue, boot verification may be refusing system to boot, flash vbmeta with disabled verification and check if it boots.
